app/assets/stylesheets/article-show.scss

```diff
@@ -1,10 +1,6 @@
 @import 'variables';
 @import 'mixins';
 
-.stories-show {
-  @include themeable(background, theme-background, $lightest-gray);
```

Comment:
I found bunch of cases like this, where styling was kinda redundant and unnecessary.